lib/mutant/mutator/node/begin.rb

```diff
@@ -13,23 +13,11 @@ class Begin < self
         #
         # @return [undefined]
         def dispatch
-          Util::Array.each(children, self, &method(:emit_child_subset))
           children.each_with_index do |child, index|
             mutate_child(index)
-            emit(child) unless children.eql?([child])
+            emit(child) unless children.one?
           end
         end
```

Comment:
Tiny point, but do you want there to be a blank line before the class/module `end`?